src/test/kotlin/no/nav/syfo/infrastructure/identhendelse/IdenthendelseServiceSpek.kt

```diff
@@ -0,0 +1,108 @@
+package no.nav.syfo.infrastructure.identhendelse
+
+import no.nav.syfo.ExternalMockEnvironment
+import no.nav.syfo.UserConstants
+import no.nav.syfo.domain.VurderingType
+import no.nav.syfo.generator.generateIdenthendelse
+import no.nav.syfo.generator.generateVurdering
+import no.nav.syfo.infrastructure.database.dropData
+import no.nav.syfo.infrastructure.database.repository.VurderingRepository
+import no.nav.syfo.infrastructure.kafka.identhendelse.IdenthendelseService
+import org.amshove.kluent.shouldBeEmpty
+import org.amshove.kluent.shouldNotBeEmpty
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+private val aktivIdent = UserConstants.ARBEIDSTAKER_PERSONIDENT
+private val inaktivIdent = UserConstants.ARBEIDSTAKER_3_PERSONIDENT
+
+class IdenthendelseServiceSpek : Spek({
+    describe(IdenthendelseService::class.java.simpleName) {
+        val externalMockEnvironment = ExternalMockEnvironment.instance
+        val database = externalMockEnvironment.database
+        val vurderingRepository = VurderingRepository(database = database)
+        val identhendelseService = IdenthendelseService(
+            vurderingRepository = vurderingRepository,
+        )
+
+        beforeEachTest {
+            database.dropData()
+        }
+
+        describe("Vurdering eksisterer for person") {
```

Comment:
Kunne man testa en case der det ligger to ulike personidenter i databasen, og så kommer det en hendelse som sier at begge de to er utdaterte, og deretter sjekke at alt har blitt oppdatert til ny? Veldig edge, men da føler jeg man har testa alt med flere updates samtidig og litt sånt.